crates/holochain/src/test_utils/consistency.rs

```diff
@@ -1,764 +1,12 @@
-) -> DatabaseResult<()> {
-    // Clear the missing hashes from the last check. This doesn't affect allocation.
-    missing_hashes.clear();
-
-    // We need to swap these hashes so we can move them into the async_reader
-    // without reallocating.
-    let expected = std::mem::replace(expected_hashes, Vec::with_capacity(0));
-    let mut missing = std::mem::replace(missing_hashes, Vec::with_capacity(0));
-
-    // Poll the vault database for each expected hashes existence.
-    let mut r = dht_db
-                .read_async(move |txn| {
-                    for hash in &expected {
-                        // TODO: This might be too slow, could instead save the holochain hash versions.
-                        let h_hash = DhtOpHash::from_k2_op(hash);
-                        let integrated: bool = txn.query_row(
-                            "
-                            SELECT EXISTS(
-                                SELECT 1 FROM DhtOp WHERE hash = :hash AND when_integrated IS NOT NULL
-                            )
-                            ",
-                            named_params! {
-                                ":hash": h_hash,
-                            },
-                            |row| row.get(0),
-                        )?;
-                        if !integrated {
-                            missing.push(hash.clone());
-                        }
-                    }
-                    DatabaseResult::Ok((expected, missing))
-                })
-                .await?;
-    // Put the data back.
-    std::mem::swap(&mut r.0, expected_hashes);
-    std::mem::swap(&mut r.1, missing_hashes);
-    Ok(())
-}
-
-/// Concurrently Gather all published op hashes and agent's storage arcs.
-#[allow(dead_code)]
-async fn gather_published_data(
-    iter: impl Iterator<Item = Stores>,
-    concurrency: usize,
-) -> StateQueryResult<Vec<PublishedData>> {
-    use futures::stream::TryStreamExt;
-    let iter = iter.map(|stores| async move {
-        let published_hashes = request_published_ops(&stores.authored_db, None)
-            .await?
-            .into_iter()
-            .map(|(l, h, _)| (l, h))
-            .collect();
-        let storage_arc = request_arc(stores.agent.clone()).await?;
-        Ok(storage_arc.map(|storage_arc| {
-            // The line below was added when migrating to rust edition 2021, per
-            // https://doc.rust-lang.org/edition-guide/rust-2021/disjoint-capture-in-closures.html#migration
-            let _ = &stores;
-            PublishedData {
-                agent: stores.agent,
-                storage_arc,
-                published_hashes,
-            }
-        }))
-    });
-    futures::stream::iter(iter)
-        .buffer_unordered(concurrency)
-        .try_filter_map(futures::future::ok)
-        .try_collect()
-        .await
-}
-
 /// Request the published hashes for the given agent.
-#[allow(dead_code)]
 pub async fn request_published_ops<AuthorDb>(
```

Comment:
Should this still be kept, or is this function now getting used elsewhere?

Reply:
Indirectly used by 75+ tests, the allow must have been added in error